Makefile

```diff
@@ -194,30 +186,31 @@ clean:
 	find . -name "*.pyc" -type f -delete
 
 ###############################################################################
-# Building the binary
+# Updating pins
 ###############################################################################
-build: bin/calico-typha
-build-all: $(addprefix sub-build-,$(VALIDARCHES))
-sub-build-%:
-	$(MAKE) build ARCH=$*
+PIN_BRANCH?=$(shell git rev-parse --abbrev-ref HEAD)
+
+define get_remote_version
+	$(shell git ls-remote http://$(1) $(2) 2>/dev/null | cut -f 1)
```

Comment:
Looks like we're now using `http` instead of `ssh`. Won't this, for example, create problems for anyone who has a private fork of this code?

Reply:
I have things setup so the Makefiles use ssh for private and http for open source. Outside of that, users can use git's `InsteadOf` directive.